mandatory/2-logic-error.js

```diff
@@ -1,16 +1,22 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  
+  return word.trim();
 }
+word = "  CodeYourFuture  ";
```

Comment:
You have successfully completed the exercise by fixing the errors 💯 

We are however defining a `word` variable 4 times in this file, reassigning its value and never actually using the variable anywhere. We can probably delete these lines :)